src/engine/rpc/ExportDatabaseRPC.h

```diff
@@ -35,21 +38,25 @@ class ENGINE_RPC_API ExportDatabaseRPC : public BlockingRPC
     virtual const std::string TypeName() const { return "ExportDatabaseRPC"; }
 
     // Invocation method
-    void operator()(const intVector &, const ExportDBAttributes &, const std::string &);
+    void operator()(const intVector &, const ExportDBAttributes *, const std::string &);
 
     // Property selection methods
     virtual void SelectAll();
 
     // Property setting methods
     void SetIDs(const intVector &ids);
-    void SetExportDBAtts(const ExportDBAttributes &);
+    void SetExportDBAtts(const ExportDBAttributes *);
     void SetTimeSuffix(const std::string &);
 
     // Property getting methods
     const intVector          &GetIDs() const;
-    const ExportDBAttributes &GetExportDBAtts() const;
+    ExportDBAttributes       *GetExportDBAtts();
     const std::string        &GetTimeSuffix() const;
 
+    ExportDBAttributes GetReturnAtts() { return returnAtts; }
+
+    ExportDBAttributes returnAtts;
```

Comment:
Given we have `GitReturnAtts()` why is `returnAtts` public?

Reply:
I was using QueryRPC as a template since it also uses return atts. I didn't delve into the why's and wherefore's. I knew the QueryRPC worked.

Reply:
If its workable, please change GetReturnAtts to a const method, like the others above it and move the definition on line 58 to below line 60 so that the member is private.